### DISCUSSION TITLE
Configurable tag colours

### DISCUSSION THREAD

**Discussion by terrapinlogo** [mealie-recipes/mealie#1220] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Give the user the option to select the colour of the tag. This would be useful for linking like tags together across different recipes.

eg.
Set the tags for protein (beef, chicken, pork, lamb, etc) to be red and the tags for carbs (potatoes, rice, pasta, noodles etc) to be yellow

The user could also decide to set a different shade of red for each of the different proteins to give a visual link but also distinguish them.

**(Optional): Suggest A Solution**
_Replace This Text: A concise description of your preferred solution. Things to address include:_
* Details of the technical implementation
* Tradeoffs made in design decisions
* Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
Tracking Here

- https://github.com/hay-kot/mealie/issues/1241

**Comment by michael-genson** [comment]
Closed since this is being tracked
